scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -228,7 +228,7 @@ public static BuildTargetResult RestorePackages(BuildTargetContext c)
         {
             var dotnet = DotNetCli.Stage0;
 
-            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--infer-runtimes", "--fallbacksource", Dirs.Corehost)
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel", "--fallbacksource", Dirs.Corehost)
```

Comment:
This was working around the issue with the NETCore App package, I'll revert it and rebase before merging.